examples/quickstart-pytorch/pyproject.toml

```diff
@@ -1,17 +1,21 @@
 [build-system]
-requires = ["poetry-core>=1.4.0"]
-build-backend = "poetry.core.masonry.api"
+requires = ["hatchling"]
+build-backend = "hatchling.build"
 
-[tool.poetry]
+[project]
 name = "quickstart-pytorch"
 version = "0.1.0"
 description = "PyTorch Federated Learning Quickstart with Flower"
-authors = ["The Flower Authors <hello@flower.ai>"]
+authors = [
+    { name = "The Flower Authors", email = "hello@flower.ai" },
+]
+dependencies = [
+    "flwr>=1.8.0,<2.0",
+    "flwr-datasets[vision]>=0.0.2,<1.0.0",
+    "torch==2.1.1",
+    "torchvision==0.16.1",
+    "tqdm==4.65.0"
+]
 
-[tool.poetry.dependencies]
-python = ">=3.8,<3.11"
-flwr = ">=1.8.0,<2.0"
-flwr-datasets = { extras = ["vision"], version = ">=0.0.2,<1.0.0" }
-torch = "2.1.1"
-torchvision = "0.16.1"
-tqdm = "4.65.0"
+[tool.hatch.build.targets.wheel]
+packages = ["."]
```

Comment:
You're missing a new line in this file

Reply:
Are you suggesting a new line at the end of this file? Just curious, what's the reasoning for it (consistency?)?

Reply:
I think this is just good practice because otherwise it might mess up the formatting in some scenarios: https://stackoverflow.com/a/5813359. Btw, this is the reason for the red circle with a minus inside it, displayed by GitHub

Reply:
@chongshenng Here is a simple example I made to showcase the difference

<img width="475" alt="image" src="https://github.com/adap/flower/assets/9378265/90ea1800-0b48-45f5-91bd-bc6b19fc9b5d">